### DISCUSSION TITLE
Add support for "symfony" project type

### DISCUSSION THREAD

**Discussion by feinersaft** [ddev/ddev#4245] [Ideas]
Hi, I would like to suggest the feature to add another project type "symfony" to ddev. I am currently trying to figure out the requirements for this project type to be made. But I don't have any Go experience, so I will try my best. What do you think?

**Comment by rfay** [comment]
Since symfony is a framework, I'm not sure what explicit things can be done to help it along. Lots of folks already use the generic 'php' project type with symfony of course. 

Docs on adding project types are at https://ddev.readthedocs.io/en/latest/developers/project-types/

Does symfony have any consistent settings that could be auto-generated? Does it have user-generated files in a particular place? Those are two of the most important things that an explicit project type can do.

**Reply by feinersaft** [reply]
Thanks for the link, I did not know there is already an instruction page for it!

I will see if it makes sense to add another project type in this case.

**Reply by feinersaft** [reply]
One question though: why is symfony different from laravel in this regard? Isn't laravel a framework, too?

**Reply by rfay** [reply]
I think Laravel is different because somebody decided to do it :) It's probably questionable also :) But if you know what ought to be done differently in Symfony as compared to the generic 'php' project type, it can sure be done!
